app/controllers/samples_controller.rb

```diff
@@ -150,12 +150,30 @@ def report_csv
   # GET /samples/1.json
 
   def show
+    amr_drug_mappings = {}
+    amr_drug_mappings["AGly"] = "aminoglycosides"
```

Comment:
Please do make this a constant in the AmrCount model and define it in a single statement